logbook-json/src/main/java/org/zalando/logbook/json/FastJsonHttpLogFormatter.java

```diff
@@ -38,59 +40,31 @@ public FastJsonHttpLogFormatter(final ObjectMapper mapper) {
         this(mapper.getFactory());
     }
 
-    @Override
-    public String format(
-            final Precorrelation precorrelation,
-            final HttpRequest request) throws IOException {
+    public FastJsonHttpLogFormatter(final ObjectMapper mapper,
```

Comment:
I guess this constructor is not covered by tests. If you follow my approach from above you get that for free, because you would actually use it then.